resolwe_bio/processes/workflows/wgbs.yml

```diff
@@ -47,6 +48,64 @@
         performance when processing a BAM file in which
         variable-length data (read, qualities, tags) do not
         otherwise need to be decoded. Default is STRICT.
+    - name: trimming
+      label: Parameters for the trimming step
+      group:
+        - name: adapters
+          label: Adapter sequences
+          type: data:seq:nucleotide
+          required: false
+          description: |
+            Adapter sequence in FASTA format that will be removed from the read.
+            This field as well as 'Seed mismatches' and 'Simple clip threshold' parameters are needed to perform Illuminacliping.
```

Comment:
Is this designation that the parameter is needed for illuminaclipping necessary? If we need this information, would having another group of parameters help?

If yes, consider clipping line width to something like 72 characters.

Reply:
It is necessary to explain that all of those fields are required in order to perform illuminaclipping as those are all set as optional. Separating the illuminaclip parameters is more in line with the original process. Do you think that in this case it is sufficient to explain which parameters are needed only in the description for adapters input?